libraries/chain/transaction.cpp

```diff
@@ -503,11 +503,12 @@ packed_transaction::packed_transaction(packed_transaction_v0&& other, bool legac
 
 packed_transaction_v0_ptr packed_transaction::to_packed_transaction_v0() const {
    const auto* sigs = get_signatures();
-   const auto* context_free_data = get_context_free_data();
-   signed_transaction strx( transaction( get_transaction() ),
+   auto& legacy = std::get<prunable_data_type::full_legacy>(prunable_data.prunable_data);
```

Comment:
Since it is possible for this to be `none` an `EOS_ASSERT` indicating unable to convert from prunable_data_type would be a better error message than the std exception that `std::get` will throw here.

Reply:
Do you think we should pass empty bytes to packed_transaction_v0 constructor if we fail to convert prunable_data_type instead?

Reply:
No. Their is no v0 representation of a pruned transaction.

Reply:
Thanks. Fixed.